istioctl/cmd/root.go

```diff
@@ -249,6 +250,10 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(softGraduatedCmd(upgradeCmd))
 	rootCmd.AddCommand(upgradeCmd)
 
+	bugReportCmd := bugreport.Cmd()
+	hideInheritedFlags(bugReportCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(bugReportCmd)
```

Comment:
Let's put this in experimental first.

Reply:
i considered it but it's not really a feature per-se but something that's supposed to help us with info gathering for issues. it will also be made available as a standalone binary for older versions.

Reply:
Why would it be a standalone binary? Either tell users to download the new istioctl (exact same work as download this standalone binary) or backport the whole thing, in my opinion.